ui/app/src/utils/auth.ts

```diff
@@ -27,8 +27,18 @@ export function setRequestForgeryToken() {
   setGlobalHeaders({'X-XSRF-TOKEN': token});
 }
 
+export function getCookie(name:string){
```

Comment:
These are more generic than auth. Don’t belong here.

Reply:
not needed anymore